### PR TITLE
added: slider component

### PR DESCRIPTION
**Pull request by harshalini**
Added slider component in Lini's component library.
Added description of each component.
Refactored some code.
Check it here ->
https://hopeful-kepler-8ee84e.netlify.app/
Screenshot (337)
Screenshot (338)

### CODE REVIEW

File: components/sliders/sliders.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+}
+#img1:checked ~ .si1 {
+    margin-left: 0;
+}
+#img2:checked ~ .si2 {
+    margin-left: -100%;
+}
+#img3:checked ~ .si3 {
+    margin-left: -200%;
+}
```
Comment (rkapoor10) [inline components/sliders/sliders.css:48]: great...using general sibling selector.

**Review by rkapoor10 — APPROVED** [review]
The code looks good to merge.

File: components/sliders/sliders.css (hunk 1 above)
Comment (harshalini) [reply components/sliders/sliders.css:48]: Thanks!

**Review by ozairkhan2810 — APPROVED** [review]
use multiples of 4 for units as suggested by vikram
rest looks good, can merge

**Comment by harshalini** [comment]
> use multiples of 4 for units as suggested by vikram rest looks good, can merge

Ok thanks!

**Pull request merged**